db/migrate/20230119023927_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] I think you should make use of author rather than user, because the foriegn key is AuthorId coming from the Author model which is an alias to User model.